fdbclient/Schemas.cpp

```diff
@@ -602,8 +602,6 @@ const KeyRef JSONSchemas::statusSchema = R"statusSchema(
             "description":"abc"
          }
       ],
-)statusSchema"
-                                         R"statusSchema(
```

Comment:
Is this change intentional?

Reply:
Yup - there's no reason to break the string literal in two here.